vendor/data/codelists/BEIS/programme_status.yml

```diff
@@ -1,46 +1,42 @@
 ---
-attributes:
-  category-codelist:
-  name: ProgrammeStatus
-  complete: '1'
 data:
-  - code: '01'
+  - code: 1
```

Comment:
This codelist has been agreed by BEIS, so I'm wary about changing the `code`s from strings to integers here, without checking first.

Reply:
@leeky I can confirm, all BEIS want is that the status are from their list - it is our decision to code them and how the coding is defined - generally speaking, I like the idea of coding the values BEIS define in the same way IATI do and that is one of the motivation for doing this work.